packages/ccd-js-gen/test/generate-wCCD.test.ts

```diff
@@ -0,0 +1,77 @@
+import * as Gen from '../src/lib.js';
+import { assertTypeChecks } from './testHelpers.js';
+
+const moduleFileName = 'wCCD';
```

Comment:
Can you please provide a header for this file explaining what the test is, what it depends on, and how to regenerate it.

Including how the wasm file was generated.

Reply:
I have added a header now